### ISSUE TITLE
Add support for PCSX2 Flatpak

### ISSUE THREAD

**Issue by LeandroStanger** [RobLoach/net.lutris.Lutris#23]

**Comment by gasinvein** [comment]
Doesn't Lutris provide its own builds of PCSX2?

**Comment by RobLoach** [comment]
Yes. Perhaps there are dependencies that PCSX2 requires that are not included? Is there an error that occurs when run?

https://lutris.net/runners

**Comment by LeandroStanger** [comment]
Let's use the Flatpak version of [PCSX2](https://github.com/flathub/net.pcsx2.PCSX2) for Lutris Flatpak

**Comment by gasinvein** [comment]
@LeandroStanger I'm not sure if it's possible to integrate PCSX2 flatpak into ours, but we can reuse their manifest and build PCSX2 as an extension for Lutris.
